installers/pgo-installer/ansible/roles/pgo-installer/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+- name: Set output directory fact
+  set_fact:
+    output_dir: "{{ ansible_env.HOME }}/.pgo/{{ pgo_operator_namespace }}/output"
+  tags: 
+    - always
+
+- name: Setup Install Container
+  tags:
+    - always
+  block:
+  - name: Create Install Namespace
+    shell: "kubectl create namespace {{ pgo_installer_namespace }}"
+    ignore_errors: yes
```

Comment:
Not sure if the ignore errors are needed here. How the ansible roles are currently setup it shouldn't ever exist unless the job is being run. After every run the ns and clusterrolebinding are cleaned up